widget-store/src/operations.ts

```diff
@@ -1,11 +1,30 @@
-import { WorkflowContext, Workflow, HandlerContext, PostApi, ArgOptional } from '@dbos-inc/dbos-sdk';
+import { WorkflowContext, Workflow, HandlerContext, PostApi, ArgOptional, configureInstance } from '@dbos-inc/dbos-sdk';
 import { ShopUtilities } from './utilities';
+import { KafkaConfig, KafkaProduceCommunicator, Partitioners, logLevel } from '@dbos-inc/dbos-kafkajs';
 export { Frontend } from './frontend';
 
 export const PAYMENT_TOPIC = "payment";
 export const PAYMENT_URL_EVENT = "payment_url";
 export const ORDER_URL_EVENT = "order_url";
 
+const kafkaConfig: KafkaConfig = {
+  clientId: 'dbos-kafka-test',
+  brokers: [`${process.env['KAFKA_BROKER'] ?? 'localhost:9092'}`],
+  requestTimeout: 100, // FOR TESTING
+  retry: { // FOR TESTING
+    retries: 5
+  },
+  logLevel: logLevel.NOTHING, // FOR TESTING
```

Comment:
Would it be better to do like logLevel.ERROR?

Reply:
That is a copypaste from somewhere.  I'll commit whatever level you want.

Reply:
Yea your guess is good as mine... I was thinking like "what would an example production app have here?"